SpringProject/src/main/java/com/gradproject2019/userConference/repository/UserConferenceRepository.java

```diff
@@ -1,10 +1,29 @@
 package com.gradproject2019.userConference.repository;
 
 import com.gradproject2019.userConference.persistence.UserConference;
+import com.gradproject2019.users.persistence.User;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+import java.util.Optional;
+
 @Repository
 public interface UserConferenceRepository extends JpaRepository<UserConference, Long> {
 
+    @Query(value =
+            "SELECT * " +
```

Comment:
Fine here, try not to write SELECT * in production code though (it can make very big result sets if joining to many tables)

Reply:
Okay thank you, Is there a better query for selecting all data of a certain type?

Reply:
Yes - your 'dto's are effectively a map between the data in a column, and the variable in the Java - so you would be need only select the columns that go into the Java Object

Reply:
`selecting all data of a certain type` do you mean like selecting all integers or something? That's not really possible

Reply:
So is SELECT * WHERE not okay? I thought that this was just selecting the columns are you are saying

Reply:
Yes its evil https://use-the-index-luke.com/blog/2013-08/its-not-about-the-star-stupid

You should get out of the habit of using it in application code. In this case it _is_ just selecting the columns you need in this case because there are only 2 columns in the table AFAIK and they are both mapped to your dao. 

This isn't about this PR really, its more for your future when you write code in real systems

Reply:
Okay thanks Ed, I will give those a read :)